TikTokApi/browser.py

```diff
@@ -83,9 +85,11 @@ def __init__(
         page.close()
 
     def get_params(self, page) -> None:
-        # self.browser_language = await self.page.evaluate("""() => { return navigator.language || navigator.userLanguage; }""")
+        # self.browser_language = await self.page.evaluate("""() => { return
+        # navigator.language || navigator.userLanguage; }""")
```

Comment:
I'd just remove these altogether.  Commented code is a smell, and if you need to bring it back, that's what git history is for :)